blog/urls.py

```diff
@@ -0,0 +1,15 @@
+from blog.models import Commentary
+from blog.views import index, PostDetailView, CommentaryCreateView
+
+from django.urls import path
+
+app_name = "blog"
+
+urlpatterns = [
+    path("", index, name="index"),
+    path("posts/<int:id>", PostDetailView.as_view(), name="post-detail"),
+    path("posts/<int:id>/create",
+         CommentaryCreateView.as_view(),
+         name="create"),
```

Comment:
The URL pattern for 'create' should also end with a trailing slash. Consider changing it to 'posts/<int:id>/create/' to follow Django's convention and ensure consistent URL patterns.